app/services/processing_service.py

```
@@ -871,6 +1275,44 @@ def _find_matching_language(
+                for file_path in relative_path.rglob(pattern):
+                    if not file_path.is_file():
+                        continue
+
+                    file_name = file_path.name
+                    if file_name in processed_files:
+                        continue
+
+                    try:
+                        content = file_path.read_text(encoding="utf-8", errors="ignore")
+                    except Exception as e:
+                        continue
```

Ruff (0.14.5)

`try`-`except`-`continue` detected, consider logging the exception

(S112)